Makefile

```diff
@@ -26,105 +26,109 @@
 # STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF
 # THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+
+subbuilddir:=$(shell uname -o|sed -e 's|/|_|g')/$(shell git branch |grep '\* ' |cut -f2 -d' ')
```

Comment:
I think windows might not like that. Can someone with mingw test ?

Reply:
I test it under win7. The sample tree is mine